elasticapm/instrumentation/packages/psycopg2.py

```diff
@@ -140,3 +131,18 @@ def call(self, module, method, wrapped, instance, args, kwargs):
                 kwargs["scope"] = kwargs["scope"].__wrapped__
 
         return wrapped(*args, **kwargs)
+
+
+def get_destination_info(host: Optional[str], port: Union[None, str, int]) -> tuple:
+    if host:
+        if "," in host:  # multiple hosts defined, take first
+            host = host.split(",")[0]
+    else:
+        host = "localhost"
+    if port:
+        port = str(port)
+        if "," in port:  # multiple ports defined, take first
+            port = int(port.split(",")[0])
```

Comment:
nit: Any particular reason you cast back to `int` here? The result is that a single port will be a `str`, but multiple ports will result in an `int`.

Reply:
good catch! It should be cast to int no matter what